src/routes/devices.ts

```diff
@@ -584,15 +584,16 @@ export const state: RequestHandler = (req, res) => {
 
 										release = depReleases[depDev.belongs_to__application[0].id];
 
-										const image = _.get(release, 'contains__image[0].image[0]');
+										const ipr = _.get(release, 'contains__image[0]');
+										const image = _.get(ipr, 'image[0]');
 										const svcInstall =
 											image == null
 												? null
 												: serviceInstallFromImage(depDev, image);
 
-										if (image != null) {
+										if (ipr != null) {
 											varListInsert(
-												image.image_environment_variable,
+												ipr.image_environment_variable,
```

Comment:
This bug must have been lurking there for a while, I don’t see it having been touched by the recent changes.

Reply:
I think it's been there since the original version but never noticed because I don't think anyone uses them (we don't expose them in the dashboard/sdk)

Reply:
Oh but the reason it became visible if you didn't realise is due to recent changes to `varListInsert` which meant that the incorrect path would cause a failure (thanks to @thgreasi's work though that I was easily able to realise why/fix it)